.github/patch_release_template.md

```diff
@@ -0,0 +1,116 @@
+* [ ] Check if *tar.xz.asc files are [here](https://download.owncloud.com/desktop/testing). If not follow the [instructions](https://github.com/owncloud/enterprise/wiki/Desktop-Signing-Knowledge)
+* [ ] Linux: add  [build targets](https://handbook.owncloud.com/release_processes/client/supported_platforms.html) @hvonreth @jnweiger
+* [ ] Branch isv:ownCloud:desktop to isv:ownCloud:desktop:client-X.Y.Z using [obs_integration](https://github.com/owncloud/administration/blob/master/jenkins/obs_integration/) the Linux packages will always land in the :testing repository - @jnweiger
+  ```obs-deepcopy-prj.sh isv:ownCloud:desktop isv:ownCloud:desktop:client-2.5.1```
+
+
+### After build succeeds
+
+* [ ] QA: [Antivirus scan](https://handbook.owncloud.com/release_processes/client/virus.html) - @HanaGemela @jnweiger 
+* [ ] Announce the new branch to community - marketing
+* [ ] Communicate the release schedule on rocket-chat #release-coordination and mailinglist release-coordination@owncloud.com. Give a high level overview of the upcoming new features, changes etc.
+* [ ] Ensure marketing is aware (marketing@owncloud.com) and prepared for the release (social, .com website, cust.communications) - 1 week before minor, 2 weeks before major (minor/major is about impact)
+* [ ] Inform Achim (ageissel@owncloud.com) and GCX that the next version will be in 1 week (gcx@owncloud.com) - marketing
+
+
+### Sprint QA
+
+#### DEV QA
+* [ ] DEV: Run [the smoke test](https://handbook.owncloud.com/release_processes/client/smoke_test.html)
+* [ ] Linux: Run [test](https://gitea.owncloud.services/client/linux-docker-install/src/branch/master/RUN.sh) with repo=https://download.opensuse.org/repositories/isv:/ownCloud:/desktop:/testing
+* [ ] Run [the smoke test](https://handbook.owncloud.com/release_processes/client/smoke_test.html)
+* [ ] Linux: Run [test](https://gitea.owncloud.services/client/linux-docker-install/src/branch/master/RUN.sh) repo=https://download.opensuse.org/repositories/isv:/ownCloud:/desktop:/testing - @jnweiger
+* [ ] Review drone results: `make test` TODO: Mac, [Lin](https://drone.owncloud.services/client/build-linux), Win? 
+* [ ] Run the tx.pl scripts on the final code tag (20181109jw: really? What does that test?) @ogoffart
+* [ ] Run smashbox (20180719 jw: FIXME: add details, how?) (ask @dschmidt, put link to smashbox results here)
+
+#### BTR QA
+* [ ] changelog testing, TO-BE-DEFINED @HanaGemela
+* [ ] small regression test, TO_BE_DEFINED @HanaGemela
+
+* [ ] If QA fails: Abort here? or rather 1) Back to Dev team, 2) goto start of "Build" section
+
+
+### After QA Approval
+
+* [ ] DEV: remove git suffix "2.6.1git" -> "2.6.1" in [`VERSION.cmake`](https://handbook.owncloud.com/release_processes/client/branch.html#version-cmake) - @TheOneRing @guruz
+* [ ] On branch sprint-1 add the release tag v2.6.1
```

Comment:
Not sure what this line is supposed to say

Reply:
Changed to: On sprint branch create a signed tag `v2.6.1` using `git tag -u E94E7B37 tagname`